File: Software/f1c100-business-card/board/f1c100-business-card/overlay/root/prime.lua
-- Function to check if a number is prime
function is_prime(n)
    if n < 2 then
        return false
    end
    for i = 2, math.sqrt(n) do
        if n % i == 0 then
            return false
        end
    end
    return true
end

-- Find and print the first 100 prime numbers
local count = 0
local num = 2
while count < 100 do
    if is_prime(num) then
        print(num)
        count = count + 1
    end
    num = num + 1
end

File: Software/simple_sbc/board/simple_sbc/overlay/root/prime.py
import math

# Function to check if a number is prime
def is_prime(n):
    if n < 2:
        return False
    for i in range(2, int(math.sqrt(n)) + 1):
        if n % i == 0:
            return False
    return True

# Find and print the first 100 prime numbers
count = 0
num = 2
while count < 100:
    if is_prime(num):
        print(num)
        count += 1
    num += 1
